src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -1012,6 +1023,35 @@ private void ShowUpdatePackages(IVsWindowFrame windowFrame, ShowUpdatePackageOpt
             packageManagerControl?.ShowUpdatePackages(updatePackageOptions);
         }
 
+        public async Task ShowReadMeMarkdownToolWindowAsync(string filePath)
+        {
+            await JoinableTaskFactory.SwitchToMainThreadAsync(DisposalToken);
+
+            var toolWindow = FindToolWindow(typeof(ReadMeMarkdownToolWindow), 0, create: true);
+
+            if (!(toolWindow is ReadMeMarkdownToolWindow markdownToolWindow))
+            {
+                return;
```

Comment:
>return; [](http://example.com/codeflow?start=16&length=7)

Should we send an error telemetry event here?

Reply:
I'd vote no since it's highly unlikely to ever have a `null` here. Adding more detail to my original thread 2 lines up.

Reply:
Silent errors are the most difficult for us to investigate, so I very much disagree with not reporting telemetry for unexpected errors.

If this branch is never expected to happen, I suggest using a "normal" cast instead `var markdownToolWindow = (ReadMeMarkdownToolWindow)toolWindow`, so we get an exception if it fails, and other error handling should take care of reporting/telemetry.

Reply:
@zivkan explicit cast exactly what I suggested in the linked comment above.